packages/wp-source/src/libraries/api.ts

```diff
@@ -41,16 +42,25 @@ class Api {
     const query = stringify(params, { arrayFormat: "bracket", encode: false });
 
     // Send request
-    return fetch(`${requestUrl}${query && "?"}${query}`);
+    return fetch(`${requestUrl}${query && "?"}${query}`).then(
+      (response: Response) => {
+        if (!response.ok) {
+          const { status, statusText } = response;
+          throw { status, statusText, isServerError: true };
```

Comment:
I guess this needs to be `ServerError` right?

Reply:
siiii 🙂